app/javascript/components/App.jsx

```diff
@@ -22,17 +22,17 @@ function App() {
         dispatch
       }}
     >
-    
-    <Router>
-      <Switch>
-        <Route path="/orgs" exact component={Organization}/>
-        <Route path="/orgs/:name/repos" exact component={Repositories}/>
-        <Route path="/orgs/:name/repos/:repo" exact component={Repository}/>
-        <Route path="/orgs/:name/repos/:repo/dependabotalerts" exact component={DependabotAlerts}/>
-        <Route path="/login" component={Login}/>
-        <Route path="/" exact component={Homepage}/>
-      </Switch>
-    </Router>
+      <Router>
+        <Switch>
+          <Route path="/docs" exact component={Docs}/>
+          <Route path="/" exact component={Organization}/>
+          <Route path="/orgs" exact component={Organization}/>
+          <Route path="/orgs/:name/repos" exact component={Repositories}/>
+          <Route path="/orgs/:name/repos/:repo" exact component={Repository}/>
+          <Route path="/orgs/:name/repos/:repo/dependabotalerts" exact component={DependabotAlerts}/>
+          <Route path="/login" component={Login}/>
+        </Switch>
+      </Router>
```

Comment:
nit: fix the indentation :D

Reply:
Is it not supposed to be indented? I did this because I couldn't tell that it was wrapped by auth context at first.

Reply:
aight it's fixed